tests/metrics/test_converters.py

```diff
@@ -131,9 +140,17 @@ def test_sync_reduce_ddp():
     tutils.set_random_master_port()
 
     worldsize = 2
-    mp.spawn(_ddp_test_fn, args=(worldsize,), nprocs=worldsize)
+    mp.spawn(_ddp_test_fn, args=(worldsize, False), nprocs=worldsize)
```

Comment:
Is the mean reduction ever tested here? Since the default is `false` and we only pass in two arguments, it seems that `reduction_mean=True` is never tested.